docsets/Zig/docset.json

```diff
@@ -10,6 +10,10 @@
         "Zig"
     ],
     "specific_versions": [
+        {
+            "version": "0.9.0",
+            "archive": "versions/0.9.0/zig.tgz"
```

Comment:
These paths are case-sensitive. Please add `versions/0.9.0/zig.tgz` or change the path to `versions/0.9.0/Zig.tgz`.